srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=7
+wrksrc="ocs-url-release-3.1.0"
+hostmakedepends="qt5-qmake qt5-svg-devel qt5-svg kdeclarative kdeclarative-devel"
+depends="qt5"
+short_desc="Install helper program for OpenCollaborationServices (ocs://)"
+maintainer="ThatGeekyWeeb <https://github.com/ThatGeekyWeeb/> <thatgeekyweeb@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://www.opencode.net/dfn2/ocs-url"
+distfiles="https://www.opencode.net/dfn2/ocs-url/-/archive/release-${version}/ocs-url-release-${version}.tar.gz https://github.com/akiraohgaki/qtil/archive/master.tar.gz"
+checksum="d8c67920359c36eb0004221e5b5695fd04cbc9e93811c372837bdb2a8c9cbcc8 cabe912cf06568a32925602f6f5374e3be22a02b705466adcff8573e181e83f1"
```

Comment:
Multi line variables should be indented with a space:

```
a="first line
 second line"
```

Reply:
one distfile and checksum per line

Reply:
I hope this is correct, I am slightly confused what that means to do
```sh
distfiles="${homepage}/-/archive/release-${version}/ocs-url-release-${version}.tar.gz
 https://github.com/akiraohgaki/qtil/archive/v0.4.0.tar.gz"
checksum="d8c67920359c36eb0004221e5b5695fd04cbc9e93811c372837bdb2a8c9cbcc8
 c1548d3f2ca5a2fc63fce60bea2c350dfcc656b03a4e00d86a9fdc0b43f22ec8"
```